design/sketch.webidl

```diff
@@ -625,19 +625,18 @@ interface GPUQueue {
     void signal(GPUFence fence, u64 signalValue);
 };
 
-// SwapChain / RenderingContext
-dictionary GPUSwapChainDescriptor {
-    GPUTextureUsageFlags usage;
-    GPUTextureFormat format;
+// SwapChain / CanvasContext
+interface GPUCanvasContext {
 };
 
-interface GPUSwapChain {
-    void configure(GPUSwapChainDescriptor descriptor);
-    GPUTexture getNextTexture();
-    void present();
+dictionary GPUSwapChainDescriptor {
+    required GPUCanvasContext context;
+    required GPUTextureFormat format;
+    GPUTextureUsageFlags usage = GPUTextureUsage.OUTPUT_ATTACHMENT;
 };
 
-interface GPURenderingContext : GPUSwapChain {
+interface GPUSwapChain {
+    GPUTexture getTexture();
```

Comment:
bikeshed: I'm still partial to getCurrentTexture. getTexture makes it sound like there's only one texture for the life of the swapchain (although if a programmer makes this error, they will probably figure it out pretty fast).